### PR TITLE
New version: xiaoyifang.GoldenDict-ng version 24.09.0

### PR DESCRIPTION
**Pull request by zengqiu**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/173326)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Signature Update Failed.
> Executable C:\Program Files\GoldenDict-ng\QtWebEngineProcess.exe returned exit code: -1073741515
> Executable C:\Program Files\GoldenDict-ng\goldendict.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

(Automated response - build 895.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 11:38:50 AM: 
> 2024-09-16 11:38:48.389 [CLI ] Package hash verification failed. SHA256 in manifest [30e8a670e80ee8c54455e83bd2b3d2981cd3e714ea973bff500912727b826aea] does not match download [1e79ae3098e0a637b4e43afd85a74fa1f3a118fe6163c63c4e383313ec4a8006]

(Automated response - build 895.)

File: manifests/x/xiaoyifang/GoldenDict-ng/24.09.0/xiaoyifang.GoldenDict-ng.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: xiaoyifang.GoldenDict-ng
+PackageVersion: 24.09.0
+InstallerType: nullsoft
+ReleaseDate: 2024-09-13
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/xiaoyifang/goldendict-ng/releases/download/v24.09.0-Release.316ec900/GoldenDict-ng-24.09.0-Qt6.7.2.exe
+  InstallerSha256: 30E8A670E80EE8C54455E83BD2B3D2981CD3E714EA973BFF500912727B826AEA
```
Comment (stephengillie) [inline manifests/x/xiaoyifang/GoldenDict-ng/24.09.0/xiaoyifang.GoldenDict-ng.installer.yaml:11]:
```suggestion
  InstallerSha256: 1E79AE3098E0A637B4E43AFD85A74FA1F3A118FE6163C63C4E383313EC4A8006
```

Comment (zengqiu) [reply manifests/x/xiaoyifang/GoldenDict-ng/24.09.0/xiaoyifang.GoldenDict-ng.installer.yaml]: Updated InstallerSha256.

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Program Files\GoldenDict-ng\QtWebEngineProcess.exe returned exit code: -1073741515
> Executable C:\Program Files\GoldenDict-ng\goldendict.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

(Automated response - build 895.)

File: manifests/x/xiaoyifang/GoldenDict-ng/24.09.0/xiaoyifang.GoldenDict-ng.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: xiaoyifang.GoldenDict-ng
+PackageVersion: 24.09.0
+InstallerType: nullsoft
+ReleaseDate: 2024-09-13
+Installers:
```
Comment (stephengillie) [inline manifests/x/xiaoyifang/GoldenDict-ng/24.09.0/xiaoyifang.GoldenDict-ng.installer.yaml:8]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Manual Validation ended with:
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**